src/main/java/mate/academy/service/order/OrderService.java

```diff
@@ -0,0 +1,25 @@
+package mate.academy.service.order;
+
+import jakarta.transaction.Transactional;
+import java.util.List;
+import mate.academy.dto.order.OrderRequestDto;
+import mate.academy.dto.order.OrderResponseDto;
+import mate.academy.dto.order.OrderUpdateRequestDto;
+import mate.academy.dto.order.item.OrderItemResponseDto;
+
+public interface OrderService {
+
```

Comment:
if you add empty line after class or interface declaration, be consistent and add everywhere, if not - remove